htdocs/install/mysql/migration/16.0.0-17.0.0.sql

```diff
@@ -113,6 +113,25 @@ ALTER TABLE llx_ticket ADD email_date datetime after email_msgid;
 
 ALTER TABLE llx_cronjob ADD COLUMN pid integer;
 
+
+ALTER TABLE llx_element_element MODIFY COLUMN sourcetype VARCHAR(64) NOT NULL;
+ALTER TABLE llx_element_element MODIFY COLUMN targettype VARCHAR(64) NOT NULL;
+ALTER TABLE llx_c_type_contact MODIFY COLUMN element VARCHAR(64) NOT NULL;
+
+CREATE TABLE llx_element_properties
```

Comment:
Oh my god no !
We work since several year to standardize code to have element = module_name = class_dir = class_name = class_file.
We should never need such a table.